### PR TITLE
added scroll to the top of page in app-component

### PR DESCRIPTION
**Pull request by KimAT25**
added scrolling to the top of page in app-component

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -9,10 +10,19 @@ import {LanguageService} from './i18n/language.service';
 export class AppComponent implements OnInit {
   title = 'GreenCityClient';
 
-  constructor(private languageService: LanguageService) {
+  constructor(private languageService: LanguageService, private router: Router) {
```
Comment (RomanDeveloperAcc) [inline src/app/app.component.ts]: Please split injections into separate rows

File: src/app/app.component.ts [hunk 2]
```diff
@@ -9,10 +10,19 @@ import {LanguageService} from './i18n/language.service';
...
   }
 
   ngOnInit(): void {
     this.languageService.setDefaultLanguage();
+    this.navigateToStartingPositionOnPage();
+  }
+
+  navigateToStartingPositionOnPage(): void {
```
Comment (RomanDeveloperAcc) [inline src/app/app.component.ts]: Please add access modifiactor (private or public)

File: src/app/app.component.ts (hunk 1 above)
Comment (KimAT25) [reply src/app/app.component.ts]: fixed

File: src/app/app.component.ts (hunk 2 above)
Comment (KimAT25) [reply src/app/app.component.ts]: fixed

File: src/app/app.component.ts [hunk 3]
```diff
@@ -9,10 +10,21 @@ import {LanguageService} from './i18n/language.service';
 export class AppComponent implements OnInit {
   title = 'GreenCityClient';
```
Comment (nabermi2020) [inline src/app/app.component.ts]: please remove unused variable

Comment (KimAT25) [reply src/app/app.component.ts]: fixed

**Comment by nabermi2020** [comment]
checked

**Pull request merged**